content/scout/integrations/registry/ecr.md

```diff
@@ -0,0 +1,211 @@
+	box User's AWS
+		participant EventBridge
+		participant SNSTopic
+		participant SecretsManager
+	  participant ECR
+	end
+	box Docker Scout
+		participant Scout API
+	end
+
+	# During setup
+  SNSTopic ->> Scout: CloudFormation Stack Event
+  Scout ->> SecretsManager: Set up Scout credentials for EventBridge
+
+	# After integration
+  User ->> ECR: Push an image
+  EventBridge -->> Scout: Push image Event
+  Scout ->> ECR: Login, and pull image
+  ECR ->> Scout: Pull image
+```
+
+After the integration, Docker Scout automatically pulls and analyzes images
+that you push to the ECR registry. Metadata about your images are stored on the
+Docker Scout platform, but Docker Scout doesn't store the container images
+themselves. For more information about how Docker Scout handles image data, see
+[Data handling](../../data-handling.md).
+
+### CloudFormation stack template
+
+The following table describes the configuration resources.
+
+> **Note**
+>
+> Creating these resources incurs a small, recurring cost on the AWS account.
+> The **Cost** column in the table represents an estimated monthly cost of the
+> resources, when integrating an ECR registry that gets 100 images pushed per day.
+
+| Resource type                 | Resource name          | Description                                                                                  | Cost  |
+| ----------------------------- | ---------------------- | -------------------------------------------------------------------------------------------- | ----- |
+| `AWS::SNSTopic::Topic`        | `SNSTopic`             | SNS topic for notifying Docker Scout when the AWS resources have been created.               | Free  |
+| `AWS::SNS::TopicPolicy`       | `TopicPolicy`          | Defines the topic for the initial setup notification.                                        | Free  |
+| `AWS::SecretsManager::Secret` | `ScoutAPICredentials`  | Stores the credentials used by EventBridge to fire events to Scout.                          | $0.42 |
+| `AWS::IAM::Role`              | `AssumeRoleScoutSetup` | This role is used during initial setup, and has write access to `ScoutAPICredentials`.       | Free  |
+| `AWS::Events::ApiDestination` | `ApiDestination`       | Sets up the EventBridge connection to Docker Scout for sending ECR push and delete events.   | $0.01 |
+| `AWS::Events::Connection`     | `Connection`           | EventBridge connection credentials to Scout.                                                 | Free  |
+| `AWS::Events::Rule`           | `Rule`                 | Defines the rule to send ECR pushes and deletes to Scout.                                    | Free  |
+| `AWS::IAM::Role`              | `InvokeApiRole`        | Internal role to grant the event access to `ApiDestination`.                                 | Free  |
+| `AWS::IAM::Role`              | `AssumeRoleEcrAccess`  | Grants the following ECR privileges to Docker Scout: login access, pull images, list images. | Free  |
```

Comment:
This role will have access to `ScoutAPICredentials`, used to setup the Scout integration.